src/renderer/bowler_hat.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from "blockly/core";
+
+export class BowlerHat extends Blockly.blockRendering.Hat {
+  constructor(constants) {
+    super(constants);
+    // Calculated dynamically by computeBounds_().
+    this.width = undefined;
```

Comment:
mmm I think this is typed as `number`, can you set it to be `0` by default?

Reply:
Done